### PR TITLE
update the DartElementPresentationUtil class

### PR DESCRIPTION
**Pull request by devoncarew**
- update the DartElementPresentationUtil class

This class is copied from the Dart plugin - this updates to the latest version.

### CODE REVIEW

File: src/io/flutter/preview/DartElementPresentationUtil.java [hunk 1]
```diff
@@ -60,7 +62,7 @@ public static Icon getIcon(@NotNull Element element) {
         if (element.isTopLevelOrStatic()) return STATIC_FIELD_ICON;
         return Field;
       case ElementKind.FUNCTION:
-        return element.isTopLevelOrStatic() ? TOP_LEVEL_FUNCTION_ICON : Function;
+        return element.isTopLevelOrStatic() ? TOP_LEVEL_FUNCTION_ICON : Lambda;
```
Comment (stevemessick) [inline src/io/flutter/preview/DartElementPresentationUtil.java:65]: Shouldn't this also check if the function has a name before using `Lambda`?

Comment (devoncarew) [reply src/io/flutter/preview/DartElementPresentationUtil.java:65]: I'm not sure here - the 'lamda' icon is the one that the Dart plugin is using for non-top level functions, whether the item is actually a lamda or not.

Comment (devoncarew) [reply src/io/flutter/preview/DartElementPresentationUtil.java:65]: Note that this PR is just updating our code to the latest from the Dart plugin - if we think we should change something here we should adjust it upstream.

**Review by stevemessick — APPROVED** [review]
Sorry, I thought I'd already approved this one.

**Pull request merged**